<launch>

  <!-- Joy node -->
  <node pkg="joy" type="joy_node" name="joy_node" output="screen">
    <param name="dev" value="/dev/input/js0" type="string "/>
    <param name="deadzone" value="0.05" type="double"/>    
    <param name="autorepeat_rate" value="20" type="double"/> 
  </node>
 
  <node pkg="transfor_joy" type="transfor_joy" name="transfor_joy" output="screen">
  	 <param name="l_scale_1" value="90" type="double"/> 
  	 <param name="w_scale_1" value="80" type="double"/> 
  	 <param name="l_scale_2" value="150" type="double"/> 
  	 <param name="w_scale_2" value="130" type="double"/> 
      <remap from="/cmd_vel" to="/ws2500_chassis/cmd_vel"/>
  </node>

</launch>
